lib/ansible/module_utils/urls.py

```diff
@@ -1787,6 +1793,20 @@ def prepare_multipart(fields):
 #
 # Module-related functions
 #
+def set_multipart_encoding(encoding="base64"):
+    """Takes an string with specific encoding type for multipart data.
+    Will return reference to function from email.encoders library.
+    If given string is empty or related key doesn't exist
+    it will resolve to default base64 based encoding"""
+    from email import encoders
+    encoders_dict = {
+        "base64": encoders.encode_base64,
+        "7or8bit": encoders.encode_7or8bit
+    }
+    if encoders_dict.get(encoding):
+        return encoders_dict.get(encoding)
+    else:
+        return encoders_dict.get("base64")
```

Comment:
This should raise a `ValueError` if an encoder is specified that isn't supported here.